src/components/common/PickerScreen/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, TouchableOpacity, ViewPropTypes } from 'react-native';
+import Picker from 'react-native-picker';
+import styles from './styles';
+
+class PickerScreen extends Component {
+  constructor(props) {
+    super(props);
+    const { selected } = props;
```

Comment:
Just a comment, no need to change: since you passed `props` to `super` you can then use `this.props` if you want, that's why you do `super(props)`, no other reason, maybe for future compatibility.